src/main/java/forestry/apiculture/items/ItemScoop.java

```diff
@@ -49,4 +61,17 @@ public boolean mineBlock(ItemStack stack, Level world, BlockState blockState, Bl
 
 		return true;
 	}
+
+	@Override
+	public InteractionResult interactLivingEntity(ItemStack stack, Player player, LivingEntity interactionTarget, InteractionHand usedHand) {
+		if(interactionTarget instanceof Bee){
```

Comment:
Everything inside this if statement needs to be guarded by an `if (!level.isClientSide)` check. Return `InteractionResult.sidedSuccess(level.isClientSide)` outside of that check. Look at `SaddleItem` for an example of what I mean